src/points/types.ts

```diff
@@ -35,7 +35,12 @@ export type PointsMetadata = {
   }>
 }
 
-type ClaimActivityId = 'create-wallet' | 'swap'
+const claimActivities = ['create-wallet', 'swap'] as const
+type ClaimActivityId = (typeof claimActivities)[number]
+
+export function isClaimActivityId(activity: unknown): activity is ClaimActivityId {
```

Comment:
nit: i wonder if this should just be `isActivityId` (mainly because i don't want to go back and rename activity -> claimActivity everywhere 😅) or maybe `isSupportedActivityId`?